### ISSUE TITLE
Pseudo language option should only be shown to user in developer/sandbox environments

### ISSUE THREAD

**Issue by SpaceWhite** [omegaup/omegaup#1891]
## Current behaviour
In `omegaup/profile/edit` pseudo language is available

## Expected behaviour
This option should be disable because it is just an i18n identifier

**Comment by lhchavez** [comment]
It's still useful for developers and for people trying to report bugs while reproing in https://sandbox.omegaup.com

**Comment by heduenas** [comment]
So we should show it in local environments and sandbox, but not in prod.

**Comment by dgarvit** [comment]
I agree with you @heduenas 
So are we finally planning to implement this feature? I would like to work on this

**Comment by dgarvit** [comment]
Okay, so my approach to this was to check if `OMEGAUP_DEVELOPMENT_MODE` is `true`, and if it is, then make pseudo language option available.

First I tried injecting php code directly into the template, but smarty does not allow it unless a setting is enabled. And php code in templates is anyways not a good practice.

So I edited the controller, and created a dictionary of the entries in the php view, and assigned it to smarty. It is rendering as expected, except one error. The template in such cases does not parse words like `{#wordsSpanish#}`. Any solution to it? Here is how I defined the array in php view:
```
$languages = [
	'es' => "{#wordsSpanish#}",
	'en' => '{#wordsEnglish#}',
	'pt' => '{#wordsPortuguese#}',
];
```
